### DISCUSSION TITLE
Adding additional home networks after the setup of SO is completed.

### DISCUSSION THREAD

**Discussion by rafaelaraya** [Security-Onion-Solutions/securityonion#8042] [General] [answered]
Hello there,

Is it possible to add additional networks after the setup of SO is completed? the system is running fine but I just realized  I have another couple of networks I would like to add to the monitoring.  Any help is greatly appreciated. 

Rafael~

**Comment by xfaith** [comment, marked as answer]
I changed my $HOME_NET that was being monitored from between router/switch to between router/cable modem.  This required that I added the Cable Modems IP address to my HOME_NET.   
I used the documentation from the website:  https://docs.securityonion.net/en/2.3/homenet.html?highlight=HOMENET#homenet. 
If you are talking about adding additional sensors to monitor those networks well that could be: 
https://docs.securityonion.net/en/2.3/so-monitor-add.html?highlight=add%20sensor

Hope that helps,

**Comment by rafaelaraya** [comment]
Thank you very much!

Let me take a crack at it.




***@***.***
Point West Credit Union | Direct: 503.546.5387 | Fax: 503.546.5010
www.pointwestcu.com<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.pointwestcu.com%2F&data=04%7C01%7Craraya%40pointwestcu.com%7Ca842580693034938db2208d8a8370129%7C9b33c900df6b4c20a9fd9960d8c83f5b%7C1%7C0%7C637444301505956119%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=JJTiOzMnFR6EmbUbxKId3Nj%2FHzhpuXhbYjCV%2BsfO76o%3D&reserved=0>

This email and any files transmitted with it may contain confidential information and is for the sole use of the intended recipient. Any review or distribution by others is strictly prohibited. If you are not the intended recipient, please contact the sender immediately and delete the email and any attachments from your computer. Point West Credit Union specifically disclaims any responsibility or liability for any personal information, or opinions of the author, expressed in this email. All email sent to this address will be received by Point West Credit Union's corporate email system and is subject to archiving and review by someone other than the recipient.

From: David Decker ***@***.***>
Sent: Monday, May 30, 2022 6:07 PM
To: Security-Onion-Solutions/securityonion ***@***.***>
Cc: rafael x. araya ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL]Re: [Security-Onion-Solutions/securityonion] Adding additional home networks after the setup of SO is completed. (Discussion #8042)


I changed my $HOME_NET that was being monitored from between router/switch to between router/cable modem. This required that I added the Cable Modems IP address to my HOME_NET.
I used the documentation from the website: https://docs.securityonion.net/en/2.3/homenet.html?highlight=HOMENET#homenet<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.securityonion.net%2Fen%2F2.3%2Fhomenet.html%3Fhighlight%3DHOMENET%23homenet&data=05%7C01%7Craraya%40pointwestcu.com%7C932224bb0e9e47bf227508da42a1eb8d%7C9b33c900df6b4c20a9fd9960d8c83f5b%7C1%7C0%7C637895560460478569%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=fsUgg08QMaD4sm3NutjvmG68sUxzT3xU5mRpyInokio%3D&reserved=0>.
If you are talking about adding additional sensors to monitor those networks well that could be:
https://docs.securityonion.net/en/2.3/so-monitor-add.html?highlight=add%20sensor<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.securityonion.net%2Fen%2F2.3%2Fso-monitor-add.html%3Fhighlight%3Dadd%2520sensor&data=05%7C01%7Craraya%40pointwestcu.com%7C932224bb0e9e47bf227508da42a1eb8d%7C9b33c900df6b4c20a9fd9960d8c83f5b%7C1%7C0%7C637895560460478569%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LIfNqYAdkatk43yGFT3beTvGc9wh5xS%2FZ6cBVx0knOg%3D&reserved=0>

Hope that helps,

-

**Reply by rafaelaraya** [reply]
Hi there,

After inspecting the global.sls file I did add the range of 10.17.0.0/16 but when looking at the alerts I am only seeing alerts from one of my VLANs (10.17.9.0/24) -- I do not see anything for 10.17.8.0/24 or 10.17.10.0/24 where I have other devices.  Do I need to add those VLANs separately on the global file?  Thanks.

**Comment by xfaith** [comment]
I would recommend looking at Kibana, and see if you  see any of that traffic say in your Zeek Logs.  
What does your full home_net look like in the global.sls.?
